language/libra-tools/libra-events-fetcher/src/main.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use libra_events_fetcher::LibraEventsFetcher;
+use libra_types::account_address::AccountAddress;
+use structopt::StructOpt;
+use tokio;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// Full URL address to connect to - should include port number, if applicable
+    #[structopt(short = "u", long)]
+    url: String,
+    #[structopt(subcommand)] // Note that we mark a field as a subcommand
+    cmd: Command,
+}
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    #[structopt(name = "get-payment-events")]
+    GetPaymentEvents { accounts: Vec<AccountAddress> },
+}
+
```

Comment:
#[tokio::main]
async fn main() {


then you don't need create runtime and use await / join future instead